dispatcher/config.ini

```diff
@@ -71,14 +71,24 @@ MasterDAQConfig = {
         }
     }
 
-# Addresses for the VME crates
-VMEConfig = {
-  "0": "192.168.131.60",
-  "1": "192.168.131.61",
-  "2": "192.168.131.62",
-  "3": "192.168.131.63",
-  "4": "192.168.131.64"
+# Adresses for VME and NIM crates:
+CrateConfig = {
+  'vme': {
+    "0": "192.168.131.60",
+    "1": "192.168.131.61",
+    "2": "192.168.131.62",
+    "3": "192.168.131.63",
+    "4": "192.168.131.64"
   }
+  'nim': {
+    "0": "192.168.131.90",
+    "1": "192.168.131.91",
+    "2": "192.168.131.92",
+    "3": "192.168.131.93",
+    "4": "192.168.131.94",
+    "5": "192.168.131.95"
```

Comment:
This does not include any of the NV/MV crates? Perhaps it is worth a comment in the config

Reply:
> Right, for bookkeeping it might be nice to add some description of the PR wink
> 
Totally right. Sorry. I will add descriptions next time. Let me add some for this one now:

The only change for the dispatcher is that it commands the Hypervisor to look for commands in the db and execute them, now. The actual point where this bit is called in the dispatcher's event loop is probably the most critical bit. At the end of the loop, all the changes depending on the current status should have been executed already, that's why mostly Darryl (but I agree, as far as I can tell), think that either the end of the loop or the beginnig (which is more or less the same spot) would be the best place to put it. Do you see a reason why this wouldn't be a good idea?

In the Hypervisor (mentioned daqnt PR) we now want to have access to NIM IPs, also. That's why we extended the config. Darryl wanted a nested dict for that. I hope I found all the places, where we thus require to ask for the VME part of the nested dict, instead of using the entire dict, as before. I also changed the naming to be consistent with what the object holds.

Reply:
What kind of comment would you like?
In some nuclear restart option, I think all the crates (so far only VME) are shut down and restarted. So every crate we add here will be affected by this. So that's probably at least one thing we should consider before adding more crates here. Maybe we want this even.